### PR TITLE
Add 'join table' for visits PR 

### PR DESCRIPTION
**Pull request by kkasaija**
### Milestone reached
- Added vets, visits, and specialties tables.
- Inserted data for specialties.
- Inserted data for visits.
- Ran queries as shown in the attached screenshots.
## Screenshots
Capture1
Capture2
Capture3
Capture4

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -72,4 +72,85 @@ SELECT animals.name, animals.escape_attempts FROM animals
 JOIN owners ON animals.owners_id= owners.id 
 WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 SELECT owners.full_name, COUNT(animals.name) FROM owners 
-JOIN animals ON animals.owners_id = owners.id GROUP BY owners.full_name ORDER BY count DESC;
\ No newline at end of file
+JOIN animals ON animals.owners_id = owners.id GROUP BY owners.full_name ORDER BY count DESC;
```
Comment (tufoinnkuo10) [inline queries.sql:75]: - [ ] Great job with the Queries. Please kindly add screenshots of your query results to the pull request description. this help the reviewer or viewer check them directly if the queries match

File: data.sql [hunk 2]
```diff
@@ -51,3 +51,335 @@ UPDATE animals SET owners_id = 2 WHERE name='Gabumon' OR name='Pikachu';
...
+
+-----------------------------------------------------------------------
+/*PROJECT FOUR*/
+
+INSERT INTO vets(name,age,date_of_graduation)
+VALUES('William Thatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
```
Comment (tufoinnkuo10) [inline data.sql:63]: - [ ] please kinldy add data to the `specializations` table. you missed that part. 
nice work

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ Descriptive pull request message



Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: data.sql (hunk 2 above)
Comment (kkasaija) [reply data.sql:63]: Noted, with thanks.

File: queries.sql [hunk 3]
```diff
@@ -72,4 +72,85 @@ SELECT animals.name, animals.escape_attempts FROM animals
...
+
```
Comment (SajjadAhmad14) [inline queries.sql:76]: Good job so far! You have added screenshots of query resuls in project directory but the requirement is to add them in PR description. Kindly add them so the next reviewer can check the results.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ken,
### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
PR has a title and description. :heavy_check_mark:
## To Improve 🙏
- [ ] [ OPTIONAL ] Please use [this](https://github.com/microverseinc/readme-template) new readme tempalate from Microverse in your current and all upcoming projects.
 
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 3 above)
Comment (kkasaija) [reply queries.sql:76]: Noted, Thanks

**Review by mwanawabangona — APPROVED** [review]
Hi @Kasaija-Kenneth 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkasaija** [comment]
Thank-you @mwanawabangona

**Pull request merged**